lib/node_modules/@stdlib/math/base/special/acoth/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@stdlib/math/base/special/acoth",
+  "version": "0.0.0",
+  "description": "Compute the inverse hyperbolic cotangent.",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "The Stdlib Authors",
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "directories": {
+    "benchmark": "./benchmark",
+    "doc": "./docs",
+    "example": "./examples",
+    "lib": "./lib",
+    "scripts": "./scripts",
+    "test": "./test"
+  },
+  "scripts": {},
+  "homepage": "https://github.com/stdlib-js/stdlib",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/stdlib-js/stdlib.git"
+  },
+  "bugs": {
+    "url": "https://github.com/stdlib-js/stdlib/issues"
+  },
+  "dependencies": {},
+  "devDependencies": {},
+  "engines": {
+    "node": ">=0.10.0",
+    "npm": ">2.7.0"
+  },
+  "os": [
+    "aix",
+    "darwin",
+    "freebsd",
+    "linux",
+    "macos",
+    "openbsd",
+    "sunos",
+    "win32",
+    "windows"
+  ],
+  "keywords": [
+    "stdlib",
+    "stdmath",
+    "mathematics",
+    "math",
+    "math.acoth",
+    "acoth",
+    "hyperbolic",
+    "inverse",
+    "cosine",
+    "cos",
+    "arc",
+    "arccosine",
+    "trig",
+    "trigonometry",
+    "radians",
+    "angle",
+    "polyfill",
```

Comment:
This can be removed, as this package does not serve as a polyfill, given that nothing at the standards level is happening in order to add built-in support for `acoth`.